### ISSUE TITLE
100g fr interface config on the dut side

### ISSUE THREAD

**Issue by LimeHat** [openconfig/featureprofiles#2977]
(moving this conversation from #2953 )
In some tests, there's an unusual configuration performed on DUT only for 100G FR ports, e.g.:
https://github.com/openconfig/featureprofiles/blob/91796ca845a55876969dec6e9a574c5cc0800738/feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/aggregate_all_not_forwarding_viable_test.go#L347-L351

It is unclear why is this required.

@dplore suggested that it might be related to the FEC config on the OTG side (https://github.com/search?q=repo%3Aopenconfig%2Ffeatureprofiles+%22Disable+FEC+for+100G-FR+ports+because+Novus+does+not+support+it%22&type=code), but the parameters do not match:

1. OTG config only disables the fec, while the autonegotiation is still on

https://github.com/openconfig/featureprofiles/blob/22dc0891eabdcf9b9b7bcf11a216c3d734f8f120/feature/gribi/otg_tests/encap_frr/encap_frr_test.go#L898-L903

2. DUT disables autonegotiation completely

**Comment by dplore** [comment]
@Ankur19  and @liulk do you have some experience with this issue?

**Comment by LimeHat** [comment]
A case in point: looks like it caused an issue for another vendor in #2984

**Comment by dplore** [comment]
@LimeHat are you suggesting the that the regular expectation should be auto-negotiation should work and the deviation should be to disable auto-negotiation and set duplex and port speed?

**Comment by LimeHat** [comment]
Yes, I believe so. I'm not aware of any IEEE limitations that would prevent auto-negotiation from working with FR optics.

**Comment by LimeHat** [comment]
Bumping this up. Any additional feedback? Is there an owner of this code who _needs_ it? If not, I'll open a PR to remove it.

**Comment by dplore** [comment]
no additional comments from me.  It seems ok to remove.  If you send the PR we'll run it to confirm
